src/python_pachyderm/mixin/pfs.py

```diff
@@ -605,12 +628,16 @@ def get_file(self, commit, path, URL=None):
         * `commit`: A tuple, string, or `Commit` object representing the
         commit.
         * `path`: A string specifying the path of the file.
+        * `tag`: A string to distinguish files by.
+        * `URL`: A string that specifies an object storage URL that the file
+        will be uploaded to.
         """
         res = self._req(
             Service.PFS,
             "GetFileTAR",
             req=pfs_proto.GetFileRequest(
-                file=pfs_proto.File(commit=commit_from(commit), path=path), URL=URL
+                file=pfs_proto.File(commit=commit_from(commit), path=path, tag=tag),
+                URL=URL,
             ),
         )
         return PFSFile(FileTarstream(res))
```

Comment:
As you mentioned, maybe we can just use `BytesIO` instead of `FileTarstream`.

Reply:
+1